src/dialog-editor/services/modal/modalService.ts

```diff
@@ -81,6 +85,40 @@ class ModalController {
             }
           );
         }
+        // set modal title
+        if (!this.modalData.dynamic) {
+          var titleLabel;
```

Comment:
use `let` instead of `var`

Reply:
I'll fix this one in next PR together with [other changes](https://github.com/ManageIQ/ui-components/pull/139#discussion_r137289809)